packages/db/config/IDbConfig.ts

```diff
@@ -0,0 +1,27 @@
+export interface IDbConfig {
+  automigrate: boolean;
+  renameInsteadOfDrop: boolean;
+  viewSchemaName: string;
+  appClient: IDbAppClientConfig;
+  general: IDbGeneralPoolConfig;
+}
+
+export interface IDbAppClientConfig {
+  database: string;
+  host: string;
+  user: string;
+  password: string;
+  port: number;
+  ssl: boolean;
+}
+
+export interface IDbGeneralPoolConfig {
+  database: string;
+  host: string;
+  user: string;
+  password: string;
+  port: number;
+  ssl: boolean;
+  totalMax: number;
```

Comment:
`totalMax` is a feature of the db-auto-scaling, so the setting should be configured through that package. However the normal pg-pool has a `max` setting, that should be configured through the DB module.

Reply:
The PgPool `max` is set [in the DbGeneralPool](https://github.com/fullstack-build/fullstack-one/blob/60a308de703e212a6785f1b3f308dfc220d19d9d/packages/db/lib/DbGeneralPool/index.ts#L90) using the calculated connectionsPerInstance. So iit is valid to keep those configs.